ngrinder-frontend/src/js/components/perftest/detail/Config.vue

```diff
@@ -409,8 +410,15 @@
                 return;
             }
 
+            const scriptName = this.test.config.scriptName;
+            const scriptRevision = this.test.config.scriptRevision;
             this.syncGitHubConfigRevision();
-            this.setScripts(this.test.config.scriptName);
+
+            this.test.config.scriptName = scriptName;
+            this.setScripts(scriptName);
+            this.$nextTick(() => {
+                this.test.config.scriptRevision = scriptRevision;
+            });
```

Comment:
There is a change about `change` event on Select2 v4.x.
In prior version, we could use `input` and `change` event seperately. So, there's a benefit to handling a value selection and handling a change of value(by clicking).
But, in the new version, we must trigger the `change` event to apply the change of value appropriately.

The code change above is for fixing select value lost by `input` -> `change` event trigger sequence.